### PR TITLE
Increase versions in package.json

### PR DESCRIPTION
**Pull request by edgarasg**
Related https://github.com/hostinger/engineering/issues/1579

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,16 +9,16 @@
...
+    "hubot-help": "1.0.1",
     "hubot-hostinger": "git+https://github.com/hostinger/hubot-hostinger.git",
     "hubot-jenkins": "^1.0.2",
-    "hubot-karma-classic": "^1.0.2",
+    "hubot-karma-classic": "^1.0.3",
     "hubot-links": "0.0.1",
-    "hubot-pubsub": "^1.0.2",
-    "hubot-redis-brain": "0.0.3",
+    "hubot-pubsub": "^1.0.6",
+    "hubot-redis-brain": "1.0.0",
```
Comment (ton31337) [inline package.json]: we don't use this at all, imo

Comment (edgarasg) [reply package.json]: i suppose this is used for redis

Comment (ton31337) [reply package.json]: Do we use Redis? I don't think so. We used Redis before the paid version for history. Now we don't. Please double-check and remove Redis and this afterward.

Comment (edgarasg) [reply package.json]: Will remove with next PR.

**Pull request merged**